src/collections/_documentation/guides/integrate-frontend/generate-first-error.md

```diff
@@ -50,18 +50,18 @@ Now that the Demo App is up and running on your local environment integrated wit
 
     ![Error Stack-trace]({% asset guides/integrate-frontend/generate-first-error-06.png @path %})
 
-    > * Notice that the stack trace is **minified**. This is usually the case with JavaScript code in a production environment.
-    > * Sentry can un-minify the code back to its readable form and display source (code) context lines in the stack frames. We will configure that in the next tutorial.
+    > * Notice that the stack trace is **minified**.  JavaScript is typically minified to reduce to the size of the source code
+    > * Sentry can un-minify the code back to its readable form and display source (code) context lines in the stack frames, which is covered in the next tutorial.
 
 ***
 
 ## Step 3: Create New Alerts
 
-You can create various alert rules per project and let Sentry know when, how, and who you want notified when errors occur in your application. **Alert rules** consist of **Conditions** and **Actions** which are performed once the associated conditions are met. For more information, see [Alerts](https://docs.sentry.io/workflow/notifications/alerts/).
+You can create various alert rules per project and let Sentry know when, how, and whom you want to notify when errors occur in your application. **Alert rules** consist of **Conditions** and **Actions**, which are performed when the associated conditions are met. For more information, see [Alerts](https://docs.sentry.io/workflow/notifications/alerts/).
 
-By default, each project is created with one initial alert rule, notifying all project team members (via email) the first time a new **issue** appears. This means that the next time the Checkout **error** occurs, you will not be notified about it.
+By default, each project is created with one initial alert rule, notifying all project team members (via email) the first time a new **issue** appears. This means that the next time the Checkout **error** occurs, the notification is not triggered since the error is not "new".
```

Comment:
```suggestion
By default, each project is created with one initial alert rule, notifying all project team members (via email) the first time a new **issue** appears. This means that the next time the Checkout **error** occurs, the notification is not triggered since the error is not "new."
```